app/assets/javascripts/govuk-component/search.js

```diff
@@ -0,0 +1,28 @@
+(function ($) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  function SearchBox($el){
+    var $searchFocus = $el.find('.js-search-input-focus');
```

Comment:
This JS feels like a generic "add focus class"  behaviour rather than something directly associated with search.